### PR TITLE
TuxMailer (Non-Microsoft Publisher)

### PR DESCRIPTION
**Pull request by tuxmailer-integration**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

Please note that we are experiencing a significant delay due to COVID delays, unanticipated team leaves, and high volume of connector submissions. 

- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Review by tuxmailer-integration — COMMENTED** [review]
TuxMailer (Non-Microsoft Publisher)

**Review by rahil1793 — APPROVED** [review]

**Review by rahil1793 — APPROVED** [review]

**Review by tuxmailer-integration — COMMENTED** [review]
TuxMailer (Non-Microsoft Publisher)

File: certified-connectors/TuxMailer/Readme.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+Validate an email whenever email is added in lists.
+
+## Obtaining Credentials
+
+API Key is required.
+
+## Known Issues and Limitations
+
+No Issues or limitations at the moment of api creation.
```
Comment (v-eisljami) [inline certified-connectors/TuxMailer/Readme.md:23]: There is no Deployment Instructions section. Please refer to this [file ](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md)for more information.

File: certified-connectors/TuxMailer/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+                  "type": "boolean",
+                  "description": "Is Catchall Domain",
+                  "title": "Is Catchall Domain",
+                  "enum": ["true", "false"]
+                },
+                "is_disposable": {
+                  "type": "boolean",
+                  "description": "Is Disposable",
+                  "title": "Is Disposable",
+                  "enum": ["true", "false"]
```
Comment (v-eisljami) [inline certified-connectors/TuxMailer/apiDefinition.swagger.json:61]: Are "true" and "false" valid values for boolean type. Please check this.

**Review by v-eisljami — COMMENTED** [review]
Only include files necessary for the connector. The image include is not a part of the connector artifacts.

File: certified-connectors/TuxMailer/Readme.md (hunk 1 above)
Comment (tuxmailer-integration) [reply certified-connectors/TuxMailer/Readme.md:23]: Deployment Intruction has been added

**Review by v-eisljami — CHANGES_REQUESTED** [review]
Please remove this image from the connector. This image isn't part of the connector and is not necessary, therefore is not needed, and it should be removed from the PR.

**Review by v-eisljami — APPROVED** [review]

**Pull request merged**